Sources/LLBuildManifest/Tools.swift

```diff
@@ -221,6 +221,7 @@ public struct SwiftCompilerTool: ToolProtocol {
 
     public var executable: AbsolutePath
     public var moduleName: String
+    public var moduleAliases: [String: String]?
```

Comment:
Would it make sense to have this as a dictionary that's usually empty or does an optional dictionary provide a semantic value?

Reply:
This comment applies throughout the code, this line was just the first place I thought of it.

Reply:
Either is ok I think. In the `func write(to:)`  it checks for nil and add the attribute only if non-nil.